tests/toolbox_image.sh

```diff
@@ -7,13 +7,16 @@ ARM_EMU_BIN="${ARM_EMU_BIN:-}"
 SYSTEM_UPDATE_ENTRYPOINT="/sbin/startup.sh"
 DISK_PREPARE_COMMAND="/sbin/prepare_disk.sh"
 
-# Test storage device parameters
-STORAGE_DEVICE_FILE="/tmp/storage_device.img"
-SECTOR_SIZE="512"
-SECTOR_TOTAL="7553024"
+# Test storage device parameters.
+# All storage start and and parameters are sector numbers. actual byte positions
+# are calculated by multiplying the sector number with the sector size.
+STORAGE_DEVICE_IMG="/tmp/storage_device.img"
+SECTOR_SIZE_B="512"
+STORAGE_END="7553024"
```

Comment:
To be consistent with the above, call it STORAGE_DEVICE_IMG_END (and sort it next to it)
also a comment  behind it, indicating the true size would be helpful